src/components/Navbar.js

```diff
@@ -39,10 +39,10 @@ export class Navbar extends Component {
                 <li className="nav-item nav-link" >
                   <a
                     className="nav-link text-white"
-                    href="/dashboard"
+                    href="/home"
```

Comment:
Shouldn't this be a `NavLink`?

Reply:
I don't think CSS changes the anchor tag's purpose at all but this looks like it's out of your scope.

Reply:
@kmunene I think that It would be better if you change this anchor tag to `Link` (a `react router` component) because the link created by this anchor will cause the full page to reload if it is clicked. This is out of the scope of this PR but I think that it is a simple bug(bad practice) that needs to be fixed.

Reply:
I think creating a separate task for this would be the right thing to do since the task description has nothing to do with the Navigation bar. This can be done together with a task say make the navigation bar look better. What do you think?